### DISCUSSION TITLE
AppControl Manager: It would be cool if there was a way to easily update the base policy GUID in a supplemental policy

### DISCUSSION THREAD

**Discussion by bladehstream** [HotCakeX/Harden-Windows-Security#438] [General]
Mucking around with base policies will cause all the references in supplemental policies to get out of sync and stop working. It would be nice if there was an easy way to mass-update the base policy GUID in supplemental policies :)

**Comment by bladehstream** [comment]
Additionally, it would be nice for the base policy to be displayed (once selected) in the supplemental policy screen full time.

**Comment by HotCakeX** [comment]
These are good suggestions, i'll implement both of them.
